project-base/storefront/pages/wishlist.tsx

```diff
@@ -35,7 +36,21 @@ const WishlistPage: NextPage<ServerSidePropsType> = () => {
 export const getServerSideProps = getServerSidePropsWrapper(
     ({ redisClient, domainConfig, t }) =>
         async (context) =>
-            initServerSideProps({ context, redisClient, domainConfig, t }),
+            initServerSideProps({
+                context,
+                redisClient,
+                domainConfig,
+                t,
+                prefetchedQueries: [
+                    {
+                        query: AdvertsQueryDocument,
+                        variables: {
+                            positionName: 'productListSecondRow',
```

Comment:
is the advert even rendered in the wishlist?

I have tried to create the advert for this position and in the wishlist is not rendered on the 15.0 nor on this branch


note: please check also other places like products by branch, products with flag

Reply:
Indeed, it is not rendered. I went only by references and didn't realize that we display the adverts only if we receive a category in the props of the ProductsList component.

Reply:
The same applies also to products by branch and products by flag.